stubs/decorator/decorator.pyi

```diff
@@ -5,7 +5,7 @@ _C = TypeVar("_C", bound=Callable[..., Any])
 _Func = TypeVar("_Func", bound=Callable[..., Any])
 _T = TypeVar("_T")
 
-def get_init(cls): ...
+def get_init(cls: type) -> None: ...
```

Comment:
This seems wrong https://github.com/micheles/decorator/blob/f6dbeeab5f60cee65397864890663977fb86e961/src/decorator.py#L46

Reply:
Good catch.